feeder/src/keystore.ts

```diff
@@ -67,7 +67,7 @@ export async function save(
     throw new Error('Key already exists by that name')
   }
 
-  const mnemonicKey = new MnemonicKey({ mnemonic })
+  const mnemonicKey = new MnemonicKey({ mnemonic, coinType: 118 })
```

Comment:
do you think we could pass this as an argument? I assume we will use standard cosmos coin type but it also could be that we follow terra's coin type.

Reply:
not in this PR but maybe just create an issue and update it in a future PR

Reply:
Changes applied